setup.cfg

```diff
@@ -22,13 +21,34 @@ keywords =
 packages =
     spotify_tensorflow
 
-[options.extras_require]
-tensorflow = tensorflow==1.9.0
-tensorflow_gpu = tensorflow-gpu==1.9.0
+[extras]
+tensorflow = 
+    tensorflow==1.9.0
```

Comment:
Should we avoid pinning versions here (leave unpinned or use bounds), and instead add bounds to all deps in `requirements.txt`?

Reply:
Currently, `pbr` is implicitly reading the `requirements.txt` and using that to populate `install_requires`.

So until we ditch `pbr`, we can't use the `requirements.txt` in the way we've discussed (i.e. reqs.txt has pinned versions, setup.py has bounds).

Also, this is not really a diff from the master branch. I've just added a line break.

Reply:
Oh right - yep, makes sense to wait until we ditch pbr.